eth/api.go

```diff
@@ -833,3 +833,31 @@ func (api *ScrollAPI) CalculateRowConsumptionByBlockNumber(ctx context.Context,
 	asyncChecker.Wait()
 	return rawdb.ReadBlockRowConsumption(api.eth.ChainDb(), block.Hash()), checkErr
 }
+
+// SetRollupEventSyncedL1Height sets the synced L1 height for rollup event synchronization
+func (api *ScrollAPI) SetRollupEventSyncedL1Height(height uint64) error {
+	rollupSyncService := api.eth.GetRollupSyncService()
```

Comment:
I think don't expose it to api, it's too dangerous

Reply:
nice catch. currently moved to "admin" namespace in [e39cdb1](https://github.com/scroll-tech/go-ethereum/pull/1044/commits/e39cdb16f2c8910ce9d84e1240552c9cb51e1e0b).